recipes/pyyaml/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "3.11" %}
+
+
+package:
+  name: pyyaml
+  version: {{ version }}
+
+source:
+  fn: PyYAML-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/P/PyYAML/PyYAML-{{ version }}.tar.gz
+  md5: f50e08ef0fe55178479d3a618efe21db
+
+build:
+  number: 0
+  script:
+    - python setup.py --with-libyaml build_ext --include-dirs="${PREFIX}/include" --library-dirs="${PREFIX}/lib"  # [unix]
+    - python setup.py --with-libyaml build_ext --include-dirs="%LIBRARY_INC% --library-dirs=%LIBRARY_LIB%         # [win]
```

Comment:
Though I must say I am a bit surprised that this only caused issues with `conda-build` on that one version of Python and went away with a downgrade. Is there some reason it didn't show up in the other cases?